examples/allennlp/allennlp_simple.py

```diff
@@ -105,11 +100,11 @@ def objective(trial):
     if DEVICE > -1:
         model.to(torch.device("cuda:{}".format(DEVICE)))
 
-    lr = trial.suggest_float("lr", 1e-5, 1e-1, log=True)
+    lr = trial.suggest_float("lr", 1e-2, 1e-1, log=True)
```

Comment:
I increased the lower bound of learning rate, as I found that model training didn't succeed at all.

Reply:
74c8a23 1e-2 -> 1e-3

Reply:
Sounds good 👍